images/hook/entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+
+echo "Assuming changeset from the envrionment: $RIG_CHANGESET"
+# note that rig does not take explicit changeset ID
+# taking it from the environment variables
+if [ $1 = "update" ]; then
+    echo "Starting update, changeset: $RIG_CHANGESET"
+    rig cs delete --force -c cs/$RIG_CHANGESET
+    echo "Creating or updating resources"
+    rig upsert -f /var/lib/gravity/resources/cassandra.yaml --debug
```

Comment:
@sofuture I don't understand here, how Cassandra data will migrate from old pods to new.